### PR TITLE
Env var cache

### PR DESCRIPTION
**Pull request by isaac-tfn**
Describe Changes
---

Made ZONE_CACHE_HOME and ZONE_TRANSLATION_CACHE environment variables so they can be altered more easily.

Task Checklist
----
- [ ] Have unittests been added (testing individual functions and their edge cases)
- [ ] Have integration tests been added (if applicable, to test modules of functionality)
- [ ] Updated RELEASE.md to reflect changes in PR
- [ ] Have new dependencies been added?
  - [ ] Have they been added to either `requirements.txt` or `requirements_dev.txt`.
  - [ ] Have they been added to `setup.cfg`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -121,7 +121,7 @@ add-ignore = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--cov=caf.base --cov-report=xml --doctest-modules -n auto"
+#addopts = "--cov=caf.base --cov-report=xml --doctest-modules -n auto"
```
Comment (MattBuckley-TfN) [inline pyproject.toml:124]: This shouldn't be commented out
```suggestion
addopts = "--cov=caf.base --cov-report=xml --doctest-modules -n auto"
```
